Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -1066,7 +1066,7 @@
 #define ENABLE_UNFAIR_LOCK 1
 #endif
 
-#if !defined(ENABLE_MEDIA_RECORDER_WEBM) && ENABLE(MEDIA_RECORDER) && ENABLE(OPUS) && ENABLE(WEB_CODECS)
```

Comment:
VideoFrame is only defined with ENABLE(VIDEO).
We could use that instead.
We might also want to update VideoEncoderActiveConfiguration.h.